components/layout/fragment.rs

```diff
@@ -1974,8 +1975,8 @@ impl Fragment {
                                         relative_containing_block_size: &LogicalSize<Au>,
                                         coordinate_system: CoordinateSystem)
                                         -> Rect<Au> {
-        // FIXME(pcwalton, #2795): Get the real container size.
-        let container_size = Size2D::zero();
+        // FIXME (mbrubeck): Get the real container size, taking vertical writing modes into account.
+        let container_size = Size2D(relative_containing_block_size.inline, relative_containing_block_size.block);
```

Comment:
Actually, we should maybe just store `relative_containing_block_size` in physical coordinates, since I don't think we ever need it in logical coordinates.  I'll try this.